### PR TITLE
Fix count subvectors for multiclass SVM for CSR data

### PR DESCRIPTION
**Pull request by PetrovKP**
1. Fix allocation for training CSR multiclass: `nSubsetVectors =classLabelsCount[nClasses - 1] + classLabelsCount[nClasses - 2];`
2. Fix vote predict for CSR multiclass - need to set offset indices between rows correctly

### CODE REVIEW

**Comment by PetrovKP** [comment]
ead01e0b-b48e-f14f-a7de-8cdcd4b723a1?

**Comment by PetrovKP** [comment]
http://intel-ci.intel.com/ead04415-8ce4-f1c0-be9a-8cdcd4b723a1?

**Comment by PetrovKP** [comment]
http://intel-ci.intel.com/ead0cab4-fe58-f112-be9a-8cdcd4b723a1?

File: cpp/daal/src/data_management/service_numeric_table.h [hunk 1]
```diff
@@ -576,12 +591,29 @@ class GetRowsCSR
...
+            }
+            const size_t * const rows = _block.getBlockRowIndicesPtr();
+            _rowOffsets[0]            = 1;
+            for (size_t i = 0; i < nRows; ++i)
+            {
+                const size_t nNonZeroValuesInRow = rows[i + 1] - rows[i];
+                _rowOffsets[i + 1]               = _rowOffsets[i] + nNonZeroValuesInRow;
+            }
+        }
     }
```
Comment (Vika-F) [inline cpp/daal/src/data_management/service_numeric_table.h:609]: We need convert _rowOffsets  only when mode != readOnly

File: cpp/daal/src/algorithms/multiclassclassifier/multiclassclassifier_predict_votebased_impl.i [hunk 2]
```diff
@@ -259,18 +259,24 @@ public:
...
         return s;
     }
 
 private:
     SubTaskVoteBasedCSR(size_t nClasses, size_t nRows, const SharedPtr<ClsType> & simplePrediction, bool & valid)
-        : super(nClasses, nRows, simplePrediction, valid)
+        : super(nClasses, nRows, simplePrediction, valid), _rowOffsets(nRows + 1)
     {}
+
+    TArrayScalable<size_t, cpu> _rowOffsets;
```
Comment (Vika-F) [inline cpp/daal/src/algorithms/multiclassclassifier/multiclassclassifier_predict_votebased_impl.i:279]: You don't need _rowOffsets here anymore. They were moved to GetRowsCSR.

File: cpp/daal/src/data_management/service_numeric_table.h (hunk 1 above)
Comment (PetrovKP) [reply cpp/daal/src/data_management/service_numeric_table.h:609]: No, for example we create sub table for read accesses and we need aligned to one base indices.

File: cpp/daal/src/algorithms/multiclassclassifier/multiclassclassifier_predict_votebased_impl.i (hunk 2 above)
Comment (PetrovKP) [reply cpp/daal/src/algorithms/multiclassclassifier/multiclassclassifier_predict_votebased_impl.i]: Fixed

**Pull request merged**